python/setup.py

```diff
@@ -21,10 +21,10 @@
 import os
 import distutils.ccompiler
 
-__version__ = '0.9.0'
-
+__version__ = os.environ.get('NGRAPH_VERSION', '0.0.0-dev')
```

Comment:
Don't use environment var to get version. cmake knows the version so use cmake configure

Reply:
There is no requirement that setup.py work independently. It must be run after cmake is run because you need to build ngraph. Please use cmake.

Reply:
That's not true. There are scenarios in which we use `setup.py` for development without running cmake at the same time. Being able to run `python setup.py develop` is very helpful.